### PR TITLE
Update SLVS config for connected mode

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**
The QP on SonarCloud updated, we need to change the auto-generated SLVS files to avoid conflicts.

### CODE REVIEW

File: analyzers/.sonarlint/SonarAnalyzer.slconfig [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "Profiles": {
     "CSharp": {
       "ProfileKey": "AWT63_RpV-oLu4FdMlyI",
-      "ProfileTimestamp": "2020-12-28T10:06:24Z"
+      "ProfileTimestamp": "2021-03-26T13:27:24Z"
```
Comment (andrei-epure-sonarsource) [inline analyzers/.sonarlint/SonarAnalyzer.slconfig:12]: @csaba-sagi-sonarsource I now realize that these are auto-generated files by SonarLint and maybe we should just ignore the SLVS folder.

WDYT?

Comment (csaba-sagi-sonarsource) [reply analyzers/.sonarlint/SonarAnalyzer.slconfig:12]: Indeed, I agree with you.

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM. In a separate PR the SLVS folder can be ignored if the others agree with it too.

File: analyzers/.sonarlint/SonarAnalyzer.slconfig (hunk 1 above)
Comment (andrei-epure-sonarsource) [reply analyzers/.sonarlint/SonarAnalyzer.slconfig:12]: the only problem is that it's referenced in each project... : `    <AdditionalFiles Include="..\..\.sonarlint\sonaranalyzer-dotnet\CSharp\SonarLint.xml" Link="Properties\SonarLint.xml" />`

**Pull request merged**